### ISSUE TITLE
Freezing on "OPPO F1s" phones

### ISSUE THREAD

**Issue by yvolk** [plusonelabs/calendar-widget#363]
As a user reported:

> Firstly, three people I know are experiencing EXACTLY the same issues. We are all using OPPO F1s phones. The problem came to our attention about a week ago. We've been using the widget for a couple of years. We all use Google Calendar.
Here are the elements of the problem:
1: The widget freezes and shows for example "tomorrow 12th January" and the event entry for the 12th, even though the date at the top of the widget is Sunday 12th January, which is correct. 
2: It doesn't appear to be syncing automatically because new entries fail to show in the widget, sometimes for several days.  
3: The manual syncing circular arrow does not function at all. 
We rebooted (turned phone off and on again) which corrected the issue but after a few minutes it went back to the unresponsive behaviour described above. 
I hope this is helpful for you to be able to either fix a possible problem with the widget app or to let me know if I am doing something wrong.

**Comment by yvolk** [comment]
Hi Alex.
1. What version of Android your phone has?
2. What version of ToDo Agenda app? Today it should be v.3.1.9
3. What Launcher app are you using? Please try if ToDo Agenda works with Nova Launcher.

**Comment by tandemalex** [comment]
Hi Yuri, 
thanks for your help. 

1: Android System Web View v 79.0.3945.116
Android Accessibility Suite v 7.2.0.220693075
2: ToDo Agenda v 3.1.9-565
3: I can't find an app that is called a "launcher" but the operating system is ColorOS.
I will try downloading Nova and get back to you shortly. Please stand by...

**Comment by tandemalex** [comment]
Hi again Yuri. 
I can't get ToDo Agenda to open in Nova, sorry. I can't get any widgets to appear on any of the screens in Nova...

**Comment by yvolk** [comment]
Please find instructions/advises on installing Nova Launcher on your phone.
That could allow us to figure out a solution for you.
Otherwise please search help from your device vendor/community. I simply don't know, what to change...

**Comment by tandemalex** [comment]
Okay Yuri, thanks for your help so far. I'll keep working on it in the next
few days and if I get it working I'll let you know.
I find it strange, though, that it has worked for years and only a week ago
it stopped.... Oh well.

Catch you soon. Cheers, Alex

**Comment by yvolk** [comment]
@tandemalex If you suspect that some recent changes broke app for you, you can always try to reinstall older versions. See e.g. the history of the first comment of the issue: 
https://github.com/plusonelabs/calendar-widget/issues/320

**Comment by tandemalex** [comment]
Hi Yuri,
again, thank you very much for your help - you give really good service!

This morning I uninstalled the app because it had not unfrozen for 3 days.
I reinstalled it and it has been working perfectly since then. I think
there must have been a bug in it somewhere. So now I don't even have to
bother with Nova, as the default launcher works fine.

Cheers and thanks, Alex

(I will post this on GitHub today to close the issue.)
